urlwatch

```diff
@@ -109,11 +111,11 @@ class NullHandler(logging.Handler):
 
 log.addHandler(NullHandler())
 
-ERROR_MESSAGE_URLS_TXT = """
-Error: You need to create a urls.txt file first.
+ERROR_MESSAGE_URLS = """
+Error: You need to create either a urls.yaml or a urls.txt file first.
```

Comment:
It doesn't make sense to have both. Therefore I'm saying the user needs to have one of them, if he has none. The message is only shown, if there is no urls file.

yaml file as the newer format is given precedence, as stated in the code. But I don't think it's helpful to say that here. To keep it simple I'm saying the user he should use one and not what I'm doing, if he thinks having two is a good idea. Of course I'm open for suggestions.

Perhaps we should think about another place to document the precedence?